onnxruntime/core/providers/nuphar/scripts/model_quantizer.py

```diff
@@ -256,8 +245,11 @@ def convert_matmul_model(input_model, output_model, only_for_scan=False, share_i
     converted_weights = {} # remember MatMul weights that have been converted, in case of sharing
     quantized_inputs = {} if share_input_quantization else None # remember quantized inputs that might be able to share between MatMuls
     for in_n in in_mp.graph.node:
-        if in_n.op_type == 'Slice' and len(in_n.input) == 1:
-            upgrade_slice_op(nf, in_n)
+        if upgrade_op(nf, in_n):
+            continue
+
+        if in_n.op_type == 'TopK' and len(in_n.input) == 1:
+            upgrade_topk_op(nf, in_n)
```

Comment:
Should be calling upgrade_op instead of upgrade_topk_op?

Reply:
yes.

---
In reply to: [322348495](https://github.com/microsoft/onnxruntime/pull/1787#discussion_r322348495) [](ancestors = 322348495)